### ISSUE TITLE
Rdio dead

### ISSUE THREAD

**Issue by ryan-nauman** [matt-h/rdio-enhancer#68]
:cry: 

http://help.rdio.com/customer/portal/articles/2228283-important-information-about-your-rdio-account

**Comment by Anzumana** [comment]
yeah feel the same.  Currently looking to how to best transfer playlist to a different service.
Spotify seems to be very much similar as far as features are concerned.

**Comment by Anzumana** [comment]
http://www.ivyishere.org/
this works well with the csv export option.
If anyone has a better solution please share :)

**Comment by jabalong** [comment]
Thanks. Could we keep a thread open here to discuss how and where people are planning to move?

Seems like importing our lists into Spotify is doable. But can we export back out of Spotify? Is there an equivalent to Rdio-Enhancer for Spotify?

The main reason I settled on Rdio, was because of Rdio-Enhancer and knowing that I could back up my music list. And sadly, Rdio closing has shown this to be a wise move!

So I just want to make sure that I have the same insurance to export my lists with whatever service I go to next.

**Comment by Anzumana** [comment]
I found https://github.com/watsonbox/exportify
To backup from Spotify.

**Comment by jabalong** [comment]
Just noticed this bit in Rdio's message:

"We’ll be in touch in the near future with more specifics, including the exact timing of Rdio's end of service, as well as information on exporting your Rdio data, including your playlists and favorites."

Will be interesting to see what they come up with for exporting. Too bad it took them going bankrupt to finally getting around to offering this requested feature. But good of them to do it, when they could have just shut down and left people in the lurch.

**Comment by jabalong** [comment]
Haven't looked though it yet, but this page looks very useful:

http://iangilman.com/rdio-exodus/

**Comment by Anzumana** [comment]
They just posted something official.
http://www.rdio.com/farewell
http://www.rdio.com/farewell/instructions/#faqs

**Comment by jabalong** [comment]
FYI, Rdio's official exporter seems to have the CSV ordered from most recently added to oldest.

I know some people had wanted some version of a "date added" feature to rdio-enhancer.

This doesn't give you the date, but at least an idea of the chronology of when you added stuff.

**Comment by jabalong** [comment]
Speaking of the date added, thanks Anzumana for the heads-up on Exportify. I tried it and the output is the same as rdio-enhancer, but with extra columns for the date added and a Spotify link (for getting back in). That's pretty good, gives me peace of mind to try Spotify as an alternative.

I found this page useful for an overview and screenshot of Exportify:
http://www.ilovefreesoftware.com/10/webware/export-spotify-playlists-pc.html

And once you're logged into Spotify in your browser, can just go to this link to run Exportify:
https://rawgit.com/watsonbox/exportify/master/exportify.html
